site/docs/contribute.md

```diff
@@ -45,6 +45,16 @@ The Iceberg community prefers to receive contributions as [Github pull requests]
 * If a PR is related to an issue, adding `Closes #1234` in the PR description will automatically close the issue and helps keep the project clean
 * If a PR is posted for visibility and isn't necessarily ready for review or merging, be sure to convert the PR to a draft
 
+### Merging Pull Requests
+
+Most pull requests can be merged once a committer is satisfied with the code in the PR. Before merging all review comments should be addressed either by making changes or agreeing the request is out of scope for the PR. For larger changes or additions to public APIs committers will wait at least 24 hours before merging to ensure there is no additional feedback from members of the community. As in all ASF governed projects committers are expected to act in the best interest of the project. If a committer feels there might be a conflict of interest with a pull request they review, they are encouraged to ask for another committer to look at the pull request.
+
+A -1 vote on a PR (left as a comment with a reason attached) indicates a reviewer does not think the PR should be merged and that immediate changes by the PR author cannot address concerns. If the author and reviewer do not agree on the -1 vote, a discussion thread should be raised on the developer mailing list to come to a consensus on a path forward. One example where a -1 vote might be used is if a change should go through the [Iceberg improvement proposal](#apache-iceberg-improvement-proposals) process. Generally, -1 votes on PRs should be rare. Requesting changes on a PR indicates the reviewer believes the PR has merit but still needs issues addressed before merging. Reviewer in this context is anyone leaving comments on a PR, they can be a contributor, a committer or a PMC member.
```

Comment:
I feel this was a part where people were not disagreeing in the bylaws document, and we have not concluded a direction yet logistically.

On one side, there are people that think merging PR is a vote, and should be described like a vote. I think as engineer this has the beauty of unifying everything under the same framework and the same set of rules. There are also a few bylaws in Apache projects that are written in that way. (e.g. ORC) 

On the other side, there are people that consider merging PR and other voting matters as fundamentally different types of things and should be discussed separately.

To me, it feels like just wording debates and in the end people are talking about the same thing anyway... JB, if the wording is saying "A disagreement" rather than "a -1 vote", would you agree with the whole sentence here?

Reply:
I updated the language @jbonofre